src/huggingface_hub/hf_api.py

```diff
@@ -646,19 +649,23 @@ def _validate_or_retrieve_token(
         function_name: Optional[str] = None,
     ):
         """
-        Retrieves and validates stored token or validates passed token.
         Args:
+        Retrieves and validates stored token or validates passed token.
             token (``str``, `optional`):
-                Hugging Face token. Will default to the locally saved token if not provided.
+                Hugging Face token. Will default to the locally saved token if
+                not provided.
             name (``str``, `optional`):
-                Name of the repository. This is deprecated in favor of repo_id and will be removed in v0.7.
+                Name of the repository. This is deprecated in favor of repo_id
+                and will be removed in v0.7.
             function_name (``str``, `optional`):
-                If _validate_or_retrieve_token is called from a function, name of that function to be passed inside deprecation warning.
+                If _validate_or_retrieve_token is called from a function, name
+                of that function to be passed inside deprecation warning.
         Returns:
             Validated token and the name of the repository.
         Raises:
-            :class:`EnvironmentError`: If the token is not passed and there's no token saved locally.
-            :class:`ValueError`: If organization token or invalid token is passed.
+            :class:`EnvironmentError`: If the token is not passed and there's no
```

Comment:
Should we also add `ValueError` on file size being greater than 30GB?

Reply:
Will be discussed in a separate issue https://github.com/huggingface/huggingface_hub/issues/995.